### ISSUE TITLE
TypeError: 'undefined' is not a function (evaluating 'this._getPreviousSiblingElement(element, null)')

### ISSUE THREAD

**Issue by gardon** [squizlabs/HTML_CodeSniffer#168]
I'm not sure where this method comes from, stack below:
```
/root/HTML_CodeSniffer/Standards/Section508/Sniffs/A.js:101
  /root/HTML_CodeSniffer/Standards/Section508/Sniffs/A.js:187
  /root/HTML_CodeSniffer/Standards/Section508/Sniffs/A.js:44
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:287
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:247
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:151
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:54
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:72
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:379
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:386
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:403
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:408
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:387
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:367
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:313
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:73
  /root/HTML_CodeSniffer/Contrib/PhantomJS/../../HTMLCS.js:55
  /root/HTML_CodeSniffer/Contrib/PhantomJS/runner.js:22
  phantomjs://webpage.evaluate():1
  phantomjs://webpage.evaluate():1
  phantomjs://webpage.evaluate():1
```
Using PhantomJS 1.9.0 and latest master

**Comment by gardon** [comment]
Someone did some copy+paste from WCAG2 AAA sniffer and didn't notice the extra methods were missing. I added these 3 and now sniffer is working:

https://github.com/squizlabs/HTML_CodeSniffer/blob/d209ce54876657858a8a01528ad812cd234f37f0/Standards/WCAG2AAA/Sniffs/Principle1/Guideline1_1/1_1_1.js#L444

https://github.com/squizlabs/HTML_CodeSniffer/blob/d209ce54876657858a8a01528ad812cd234f37f0/Standards/WCAG2AAA/Sniffs/Principle1/Guideline1_1/1_1_1.js#L494

https://github.com/squizlabs/HTML_CodeSniffer/blob/d209ce54876657858a8a01528ad812cd234f37f0/Standards/WCAG2AAA/Sniffs/Principle1/Guideline1_1/1_1_1.js#L550

**Comment by ironikart** [comment]
Resolved in f209267f2f0ed61dd56ffc08dbf09caad32c1969
